content/shmem_sync_all.tex

```diff
@@ -0,0 +1,44 @@
+\color{ForestGreen}
+\apisummary{
+    Registers the arrival of a \ac{PE} at a barrier and suspends \ac{PE}
```

Comment:
Sync doesn't provide completion guarantees for SHMEM communication routines.  Take a look at the new pipelining example in the contexts proposal.